grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJobExecutionListener.groovy

```diff
@@ -120,7 +121,8 @@ class ClientBatchJobExecutionListener implements JobExecutionListener{
 
         final String encodedContentAfterDate = URLEncoder.encode(contentAfterDate, 'utf-8')
         final String encodedPath = URLEncoder.encode(path, 'utf-8')
-        final grabPath = "http://${host}:${port}/grabbit/job?path=${encodedPath}&after=${encodedContentAfterDate}"
+        final String encodedExcludedPaths = URLEncoder.encode(excludePaths, 'utf-8')
+        final grabPath = "http://${host}:${port}/grabbit/job?path=${encodedPath}&excludePaths=${encodedExcludedPaths}&after=${encodedContentAfterDate}"
```

Comment:
May want to start looking at using something like `org.apache.http.client.utils.URIBuilder` so that edge-cases around URL encoding are taken care of (like, what if `port` is blank because it's been mapped to 80?).